lib/modules/apostrophe-areas/public/css/user.less

```diff
@@ -210,6 +244,9 @@
     margin-right: auto;
     max-width: 200px;
     .apos-rounded;
+    &.apos-widget-groups {
```

Comment:
does this selector need to be so specific?

Reply:
Yes I think so, because it's overriding another max-width expressed with a lot of specificity.